bin/ensure-kubectl

```diff
@@ -52,8 +52,11 @@ fi
 if ! hash kubectl 2>/dev/null; then
   # If being run from the CircleCI platform, install as below
   if [ -n "$CIRCLE_BUILD_NUM" ] ; then
-    cloud-login
-    gcloud --quiet components update kubectl
+    sudo /opt/google-cloud-sdk/bin/gcloud --quiet components update kubectl
+  # If being run with JenkinsCI, install as below
+  elif [ -n "$JENKINS_HOME" ] ; then
+    wget "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl" -O /var/lib/jenkins/tools/kubectl
+    chmod +x /var/lib/jenkins/tools/kubectl
```

Comment:
From a DRY perspective, can the conditional for `JENKINS_HOME` just set an install location. The `wget` command seems the same otherwise.

Reply:
👍 I agree. This is actually in conflict with #83. After that one merges, the rebase I've done locally is to make this into
```
  if [[ -d /var/lib/jenkins/tools ]]; then
    cp /usr/local/bin/kubectl /var/lib/jenkins/tools/kubectl
  fi
```